tests/integration/test_ethernet.py

```diff
@@ -106,6 +106,8 @@ def test_static_ip_with_ethernet(testnic1, provider, network_lsr_nm_mock):
             "ip": {"address": [ip_address]},
         }
     ]
+    if provider == "initscripts":
+        subprocess.check_call(["dnf", "-y", "install", "network-scripts"])
```

Comment:
I think this should be done at the ".yml" level - not sure where - in one of these places:
- tests/tests_integration_pytest.yml
- tests/playbooks/integration_pytest_python3.yml
- tests/tests_default_initscripts.yml

That way you can easily adjust the name of the package to install based on the platform, and use the `package:` module

Reply:
Sounds good, updated.